crates/host/src/wasmbus/mod.rs

```diff
@@ -1,7 +1,7 @@
-/// Wasmbus lattice configuration
+/// wasmCloud host configuration
```

Comment:
This is `wasmcloud-host` crate, so "wasmCloud" here has no meaning. It's wasmbus protocol that is implemented as is important here